designer/server/src/main/scala/pl/touk/nussknacker/ui/process/migrate/RemoteEnvironment.scala

```diff
@@ -119,6 +127,31 @@ trait StandardRemoteEnvironment extends FailFastCirceSupport with RemoteEnvironm
     } yield ()).value
   }
 
+  override def migrateScenario(localProcess: DisplayableProcess, category: String)(implicit ec: ExecutionContext, loggedUser: LoggedUser): Future[Either[EspError, Unit]] = {
```

Comment:
Why not just create one method with forwardedUserName: Option ? and left TODO: make forwardedUserName required? Is this the only difference between those two methods?

Reply:
I've refactored it a little so duplicated code was removed along with `migrateScenario`.